### PR TITLE
Add Some Basic Maintainer Issue Reply Templates

### PR DESCRIPTION
**Pull request by SudharakaP**
As per my discussion started at https://github.com/opencollective/opencollective/discussions/4308 and https://github.com/opencollective/opencollective/issues/4304#issuecomment-846712621, I just quickly created this with some standard issue reply templates. We can probably add more or alter this later as needed. 😄 

Also this is just my opinion/way of doing this; if you disagree or think that the current system in place is sufficient for now it's fine to close this and we can incorporate this kinda process later as well. I am open to ideas or suggestions. 😄

### CODE REVIEW

**Review by znarf — APPROVED** [review]
Looks good. Where are those templates used? Are they detected by GitHub somehow?

**Comment by SudharakaP** [comment]
> Looks good. Where are those templates used? Are they detected by GitHub somehow?

@znarf : They aren't detected by GitHub. I just added it there because they are specifically for GitHub saved replies. So all you got to do is just add each one of them to your saved replies (https://docs.github.com/en/github-ae@latest/github/writing-on-github/working-with-saved-replies/about-saved-replies) and within two clicks you can just reply to any issue or PR on GitHub. 😄

**Review by Betree — APPROVED** [review]
Looking good :+1:

File: .github/MAINTAINER_REPLY_TEMPLATES/WRONG_FORUM_USE_SLACK.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+We use the issue tracker to only track **Bugs**, **Feature Requests**, **Documentation**, **Projects** and **Security Vulnerabilities**.
```
[the remark this answers is not in the document]
Comment (SudharakaP) [reply .github/MAINTAINER_REPLY_TEMPLATES/WRONG_FORUM_USE_SLACK.md]: Thanks for pointing it out and the review. 👍🏽

**Pull request merged**